content/pt-br/docs/concepts/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Proxies no Kubernetes
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+Esta página descreve o uso de proxies com Kubernetes. 
+
+
+<!-- body -->
+
+## Proxies
+Existem vários tipos diferentes de proxies que você pode encontrar usando Kubernetes:
+
+
+1.  O [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - executa no pc do usuário ou em um pod
```

Comment:
Maybe you can use 'executa na máquina do usuário' instead of 'pc do usuário', what you think?